pkg/trtl/honu.go

```diff
@@ -590,23 +527,7 @@ func (h *HonuService) Cursor(in *pb.CursorRequest, stream pb.Trtl_CursorServer)
 		}
 
 		if opts.ReturnMeta {
-			// TODO: this is duplicated code with the Get method, make it a helper function.
-			msg.Meta = &pb.Meta{
-				Key:       object.Key,
-				Namespace: object.Namespace,
-				Region:    object.Region,
-				Owner:     object.Owner,
-				Version: &pb.Version{
-					Pid:     object.Version.Pid,
-					Version: object.Version.Version,
-					Region:  object.Version.Region,
-				},
-				Parent: &pb.Version{
-					Pid:     object.Version.Parent.Pid,
-					Version: object.Version.Parent.Version,
-					Region:  object.Version.Parent.Region,
-				},
-			}
+			msg.Meta = returnMeta(*object)
```

Comment:
Any particular reason why we need to dereference the Honu object here?

Reply:
Not sure exactly, I tried it without and got this as a suggested change...